managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -705,7 +709,9 @@ public void asyncAddEntry(ByteBuf buffer, AddEntryCallback callback, Object ctx)
         // Jump to specific thread to avoid contention from writers writing from different threads
         executor.executeOrdered(name, safeRun(() -> {
             OpAddEntry addOperation = OpAddEntry.createNoRetainBuffer(this, buffer, callback, ctx);
+            checkInactiveLedgerAndRollOver(lastAddEntryTimeMs);
```

Comment:
should we wrap also the `internalAsyncAddEntry` at line 724?
https://github.com/apache/pulsar/pull/13073/files#diff-f6a849bd8fdb782ef6c17a2e07a2c54c3dc7d1655c00ec3546d5f3b3fc61e970L712-R732

Reply:
yes. we can move it to avoid adding at the caller method.

Reply:
done